mediabridge/api/app.py

```diff
@@ -51,6 +52,23 @@ def search_movies() -> tuple[Response, int]:
             movies_list = [row._asdict() for row in movies]
             return jsonify(movies_list), 200
 
+    @app.route("/api/v1/movie/recommend", methods=["POST"])  # type: ignore
+    def recommend_movies() -> tuple[Response, int]:
+        data = request.get_json()
+        if not data or "movies" not in data:
+            return jsonify({"error": "JSON body with 'movies' array is required."}), 400
+        movies = data["movies"]
+        if not isinstance(movies, list) or not all(isinstance(m, str) for m in movies):
```

Comment:
We originally discussed sending in the movies as IDs, not titles. Check with @JamesKohlsRepo if we still want to do that.